packages/client/src/proxy.js

```diff
@@ -4,10 +4,28 @@ import http from 'http';
 import https from 'https';
 import logger from '@percy/logger';
 import { stripQuotesAndSpaces } from '@percy/env/utils';
+import { PacProxyAgent } from 'pac-proxy-agent';
 
 const CRLF = '\r\n';
 const STATUS_REG = /^HTTP\/1.[01] (\d*)/;
 
+// function to create PAC proxy agent
+function createPacAgent(pacUrl, options = {}) {
+  try {
+    const agent = new PacProxyAgent(pacUrl, {
+      keepAlive: true,
+      ...options,
+      rejectUnauthorized: false
```

Comment:
That is by default setting behaviour of `NODE_TLS_REJECT_UNAUTHORIZED` - ideally should be governed by this env var

Reply:
Will handle this on LCNC side since this is needed in 1 case.